app/views/documents/show/_actions.html.erb

```diff
@@ -1,24 +1,71 @@
-<ul class="govuk-list">
-  <li>
-    <%= link_to "Publish", publish_document_path(@document), class: "govuk-link" %>
-  </li>
-  <li>
-    <%= link_to "Edit document", edit_document_path(@document), class: "govuk-link" %>
-  </li>
-  <li>
-    <%= link_to "Edit associations", document_associations_path(@document), class: "govuk-link" %>
-  </li>
-  <li>
-    <%= link_to "View on GOV.UK", DocumentUrl.new(@document).public_url, class: "govuk-link" %>
-  </li>
-  <li>
-    <%= link_to "Preview", DocumentUrl.new(@document).preview_url, class: "govuk-link" %>
-  </li>
-  <li>
-    <%= link_to "Shareable preview", DocumentUrl.new(@document).secret_preview_url, class: "govuk-link" %>
-  </li>
-</ul>
+<%
+publish_button = { text: t("documents.show.actions.publish"), href: publish_document_path(@document) }
+view_live_button = { text: t("documents.show.actions.view_live"), href: DocumentUrl.new(@document).public_url }
+share_draft_button = { text: t("documents.show.actions.share_draft"), href: DocumentUrl.new(@document).secret_preview_url }
+preview_button = { text: t("documents.show.actions.view_draft"), href: DocumentUrl.new(@document).preview_url }
+create_draft_button = { text: t("documents.show.actions.create_draft"), href: edit_document_path(@document) }
+submit_2i_button = { text: t("documents.show.actions.submit_2i"), href: "submit_2i" }
+delete_draft_button = { text: t("documents.show.actions.delete_draft"), href: "delete" }
+withdraw_button = { text: t("documents.show.actions.withdraw"), href: "withdraw" }
+unpublish_button = { text: t("documents.show.actions.unpublish"), href: "unpublish" }
 
-<div class='govuk-tag' title="<%= t("publication_states.#{@document.publication_state}.description") %>">
-  <%= t("publication_states.#{@document.publication_state}.name") %>
+case @document.publication_state
+when "sent_to_draft" || "sending_to_live" || "error_sending_to_live" # Draft
+  primary_button = submit_2i_button
+  secondary_button = preview_button
+  ternary_button = publish_button
+  destructive_button = delete_draft_button
+when "sent_to_review" # Ready for review
```

Comment:
We're not anticipating sent_to_review to be part of publication state (see https://trello.com/c/NVjuv8Fs/170-the-way-we-encode-states-needs-to-be-more-flexible-so-we-can-add-more) so this case statement won't ever execute this block.

It's probably worth us having a chat to try get design/dev/product in similar mindsets regarding state.

Reply:
Would be good to discuss them. This is based on the design states Alex T. prepared for design. I can remove this case for now.